### PR TITLE
Development

### PR DESCRIPTION
**Pull request by kirbyk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,275 @@
-# Pebble Bootstrap
+Slate
+=====
+
+Slate is a front-end framework for developing Pebble mobile configuration pages.
+It's the fastest way to make a clean UI for a Pebble app's mobile configuration
+page.
+
+*Current version: [v0.0.1][dist]*
```
Comment (matthewtole) [inline README.md]: This link reference doesn't exist.

File: README.md [hunk 2]
```diff
@@ -1 +1,275 @@
...
+> Note that jQuery is also required, but not included in the framework.
+
+There are several quick ways to getting started with Slate.
+
+### Via CDN
+
+Pebble hosts both the CSS and JS files via CDN.
+
+```html
+<link href="https://somepebblecdn.com/slate/0.0.1/css/slate.min.css" rel="stylesheet">
```
Comment (matthewtole) [inline README.md]: I vote that we actually buy and use this domain.

File: README.md [hunk 3]
```diff
@@ -1 +1,275 @@
...
+```
+
+Documentation
+-------------
+
+Here is a list of the different components you can create with Slate.
+
+### Paragraphs
+
+![paragraph](http://i.imgur.com/K7gXSGm.png)
```
Comment (matthewtole) [inline README.md]: A lot of places block imgur. Is there somewhere better we can host these files? In the repo on the `gh-pages` branch maybe?

Comment (kirbyk) [inline README.md]: Yeah, I plan on storing everything on master. I did this temporarily.

File: README.md (hunk 2 above)
Comment (kirbyk) [inline README.md]: Haha, I'll talk to @aheckmann and see what he says.

File: README.md (hunk 1 above)
Comment (kirbyk) [inline README.md]: Yup, soon it will though :smile:

**Pull request merged**